external-crates/move/crates/move-analyzer/src/analysis/parsing_analysis.rs

```diff
@@ -195,6 +195,14 @@ impl<'a> ParsingAnalysisContext<'a> {
         assert!(self.current_mod_ident_str.is_none());
         self.current_mod_ident_str = Some(mod_ident_str.clone());
 
+        if mod_use_defs.get(&mod_ident_str).is_none() {
```

Comment:
This did not need handling before as we were doing compilation with pre-compiled libraries right from the start. Now we do the first. Now we do first compilation without pre-compiled libraries as explained in the PR description.